WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -679,7 +679,7 @@ def stripe_token_id(options = {})
           card: {
             number: "4242424242424242",
             exp_month: 12,
-            exp_year: 2017,
+            exp_year: 2030,
```

Comment:
Possibly insane future proofing, but we `Date.today.year + 1` feels a little less arbitrary to me. Although I would love to run into this problem again in 2031 =) Whichever you prefer!